src/components/Login/utils.ts

```diff
@@ -20,8 +20,8 @@ export const decodeQr = (code: string): DecodedQrResponse => {
       throw new Error(ERROR_MESSAGE.AUTH_FAILURE_INVALID_FORMAT);
     return { endpoint: parsedCode.endpoint, key: parsedCode.key };
   } catch (e) {
-    if (e.message.includes("Unexpected token"))
-      throw new Error(ERROR_MESSAGE.AUTH_FAILURE_EXPIRED_TOKEN);
+    if (e.message.includes("token") || e instanceof SyntaxError)
+      throw new Error(ERROR_MESSAGE.AUTH_FAILURE_INVALID_FORMAT);
     throw e;
```

Comment:
just curious, what other types of errors would be thrown here?

Reply:
its mostly SyntaxError when calling `decodeQr`!